pages/trainer/[name]/index.vue

```diff
@@ -123,4 +179,10 @@ export default {
   display: block;
   margin-bottom: 0.25rem;
 }
+
+.gamify-item:hover :deep(img) {
```

Comment:
```suggestion
.gamify-item:hover > img {
```

ここはディープセレクタは不要だと思います（子孫セレクタにするかは期待するマークアップ構造が明確になるので私はそうしますがおまかせします）

Reply:
修正しました